REQUIREMENTS.txt

```
@@ -1 +1 @@
-pydantic >= 1.5, == 1.*
\ No newline at end of file
+pydantic == 1.*
```

If the goal is to have a minimum version and remain in the 1.x version, the better way to do it would be:
```suggestion
pydantic >= 1.5, < 2
```

---

Sounds good. Thanks for the suggestion.